packages/xod-client/src/workers/wasm.worker.js

```diff
@@ -92,6 +103,9 @@ _self.onmessage = e => {
       // eslint-disable-next-line no-undef
       wasmInstance = new Module(opts);
       wasmInstance.Serial = Serial;
+      wasmInstance.Time = Time;
+      // Make Time updating each millisecond
+      setInterval(() => wasmInstance.Time.set(wasmInstance.Time.get() + 1), 1);
```

Comment:
Isn't `wasmInstance.Time.set(wasmInstance.Time.get() + 1)` the same as `wasmInstance.Time.value++`?

Reply:
👌